botorch/models/fully_bayesian.py

```diff
@@ -904,7 +904,7 @@ def construct_inputs(
 
 
 class FullyBayesianSingleTaskGP(AbstractFullyBayesianSingleTaskGP):
-    r"""A fully Bayesian single-task GP model with the SAAS prior.
+    r"""A fully Bayesian single-task GP model with the Hvarfner prior.
```

Comment:
We don't really talk about priors in other high-level model descriptions. Let's also adjust the wording below (can't suggest on non-modified code directly) to make clear what the dimension-scaling refers to:

> A model with a Matern-5/2 kernel and dimension-scaled priors on the hyperparameters from  [Hvarfner2024vanilla]_  is used by default.

```suggestion
    r"""A fully Bayesian single-task GP model.
```